packages/ffe-form-react/src/index.d.ts

```diff
@@ -70,7 +70,7 @@ export interface InputGroupProps extends React.HTMLAttributes<HTMLDivElement> {
     description?: string;
     label?: string | Label;
     onTooltipToggle?: (e: React.MouseEvent | undefined) => void;
-    tooltip?: boolean | string | Tooltip;
+    tooltip?: boolean | string | React.ReactNode;
```

Comment:
Isn't [booleans and strings](https://github.com/DefinitelyTyped/DefinitelyTyped/blob/master/types/react/index.d.ts#L186-L191) also `ReactNode`s? I know it works like that with regular prop-types and it seems that way in the definition of `ReactNode` as well, if I understand the syntax.

If that's the case it seems it should be enough to just say `tooltip?: React.ReactNode`.

By the way, the `instanceOfComponent` check in the propType declaration is also wrong, it should just have been `tooltip: node` [here](https://github.com/SpareBank1/designsystem/blob/develop/packages/ffe-form-react/src/InputGroup.js#L131). It's throwing errors currently and I don't think it's worked for the last few versions of React. Since you're already here that would be a nice thing to fix as well :)

Reply:
Yes, that's true. I'll fix both :)